app/views/admin/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Admin Dashboard</h1>
+
+<%= link_to 'Merchants', '/admin/merchants' %>
+<%= link_to 'Invoices', '/admin/invoices' %>
+
+<h3> Top Customers </h3>
+<ol>
+<% @top_customers.each do |customer| %>
+  <li><%= customer[0] %> - <%= customer[1] %> purchase(s)</li>
```

Comment:
this looks like you are calling 2 different customers. maybe we could call the block variable customer_name or something?